tests/test.linter.js

```diff
@@ -26,6 +26,20 @@ var fakeCheckFileExists = () => {
   });
 };
 
+class FakeIOBase {
```

Comment:
I'm tempted to generalize and extract [fake()](https://github.com/mozilla/web-ext/blob/master/tests/helpers.js#L108-L136) because it has been helpful to preserve the original interface in a fake object.